README.md

```
@@ -24,23 +30,17 @@ Visit the [proxy service on port 8000](http://localhost:8000/) to use the applic
 ## 🖥️ Application stack
 
-The services making up the coursemapper-webserver application use the following images, hosted on Docker Hub:
-
-- proxy: not published, only required for environments without built-in HTTP routing
-- webapp: [socialcomputing/coursemapper-webserver-webapp](https://hub.docker.com/repository/docker/socialcomputing/coursemapper-webserver-webapp/general)
-- webserver: [socialcomputing/coursemapper-webserver-webserver](https://hub.docker.com/repository/docker/socialcomputing/coursemapper-webserver-webserver/general)
-- coursemapper-kg-concept-map: [socialcomputing/coursemapper-webserver-coursemapper-kg-concept-map](https://hub.docker.com/repository/docker/socialcomputing/coursemapper-webserver-coursemapper-kg-concept-map/general)
-- coursemapper-kg-recommendation: [socialcomputing/coursemapper-webserver-coursemapper-kg-recommendation](https://hub.docker.com/repository/docker/socialcomputing/coursemapper-webserver-coursemapper-kg-recommendation/general)
-- coursemapper-kg-wp-pg: [socialcomputing/coursemapper-webserver-coursemapper-kg-wp-pg](https://hub.docker.com/repository/docker/socialcomputing/coursemapper-webserver-coursemapper-kg-wp-pg/general)
+The services making up the coursemapper-webserver application are released as [container images, hosted on GitHub Packages](https://github.com/orgs/ude-soco/packages?repo_name=CourseMapper-webserver). Additional third-party images used:
 - MongoDB: [mongo (official image)](https://hub.docker.com/_/mongo)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)